cindy/index.js

```diff
@@ -63,8 +63,8 @@ if(sum >= 50 && sum <= 80){
 //6. Check whether the sum of two integers is 8, or whether their difference is 8. If one of these is the case, print true.
 let integer3 = 10
 let integer4 = 2
-let sumIntegers = integer3 + integer4
-let diffintegers = integer3 - integer4
+let sumIntegers = Math.abs(integer3 + integer4)
```

Comment:
For addition you dont need to use Math.abs() just for subtraction

Reply:
Ok, put Math.abs (+) away , let me know!